lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -96,36 +119,60 @@ feature_nv_overlay_evpn:
 
 maximum_paths:
   kind: int
-  config_get_token_append: '/^maximum-paths (\d+)$/'
-  config_set_append: '<state> maximum-paths <num>'
+  cli_nexus:
+    config_get_token_append: '/^maximum-paths (\d+)$/'
+    config_set_append: '<state> maximum-paths <num>'
+  cli_ios_xr:
+    # BT: Is "maximum-paths" without specialization equal to "maximum-paths ebgp"?
+    config_get_token_append: '/^maximum-paths ebgp (\d+)$/'
+    config_set_append: ' <state> maximum-paths ebgp <num>'
   default_value: 1
 
 maximum_paths_ibgp:
   kind: int
   config_get_token_append: '/^maximum-paths ibgp (\d+)$/'
-  config_set_append: '<state> maximum-paths ibgp <num>'
+  config_set_append: ' <state> maximum-paths ibgp <num>'
   default_value: 1
 
 network:
   multiple: true
-  config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
-  config_set_append: '<state> network <network> <route_map>'
+  cli_nexus:
+    config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
+    config_set_append: '<state> network <network> <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/^network (\S+) ?(?:route-policy )?(\S+)?$/'
+    config_set_append: ' <state> network <network> <route_policy>'
```

Comment:
The set append could be common to both right? I though that was the approach we are taking where possible. Applies globally.